code/modules/hydroponics/grown/tobacco.dm

```diff
@@ -31,7 +31,7 @@
 	plantname = "Space Tobacco Plant"
 	product = /obj/item/food/grown/tobacco/space
 	mutatelist = list()
-	reagents_add = list(/datum/reagent/medicine/salbutamol = 0.05, /datum/reagent/drug/nicotine = 0.08, /datum/reagent/consumable/nutriment = 0.03)
+	reagents_add = list(/datum/reagent/oxygen = 0.05, /datum/reagent/drug/nicotine = 0.08, /datum/reagent/consumable/nutriment = 0.03)
```

Comment:
Any reason this can't be convermol?

Reply:
Convermol converts toxin to oxygen damage, right? Wouldn't that defeat the purpose of packing it with nicotine?

Reply:
damn, is this gonna make walking around outside without internals as a lavaland podperson nigh-impossible without outside assistance/good RNG?

Reply:
Other way around. Oxyloss to toxin, which isn't a terribly out of place thing for botany plants to do to you, a few other smoking plants have toxins.

Reply:
thats so sad

Reply:
trying to avoid manufactured chems in plants

Reply:
I kinda wasn't sure about that since aiuri and libital is in ambrosia vulgaris.

Reply:
> 
> 
> trying to avoid manufactured chems in plants

Maybe atropine? It is a natural alkaloid that is found in many plants in the nightshade family, which tobacco belongs to.

Reply:
my end goal is to move out direct medicinals you'd want to manufacture in the chemistry process (or attach some difficulty to plants that contain direct medicinals, my focus hasn't really been on the botany end).

Atropine could work in the interim.